docs/saltbox/inventory/index.md

```diff
@@ -42,7 +46,7 @@ When you next run the `sonarr` tag, everything that's based off that name will c
 - app data folder => /opt/BingBangBoing
 - and so on.
 
-A common use for these overrides will probably be changing the version of the docker container being used.
+A common use for these overrides but not limited to are specifiying the version of the docker container to be used.
```

Comment:
Spelling.  Should probably be "image being used" now that I think about it.

Actually, this line deliberately doesn't refer to everything that can be changed, since it's introducing a specific example of a thing that could be done and how to do it.  At the beginning it notes that all variables are fair game, and then shows one specific example [well, more now with your additions].